vignettes/convert.qmd

```diff
@@ -29,16 +29,43 @@ library(spanishoddata)
 
 ```{r}
 #| echo: false
+#| include: false
 devtools::load_all()
 ```
 
-Set the working directory if you haven't done so already, e.g. to a folder called `spanish_od_data` in your home directory:
+
+Choose where `{spanishoddata}` should download (and convert) the data by setting the `SPANISH_OD_DATA_DIR` environment variable with the following command:
+
+```{r}
+Sys.setenv(SPANISH_OD_DATA_DIR = "~/spanish_od_data")
+```
+
+The package will create this directory if it does not exist on the first run of any function that downloads the data.
+
+<details><summary>Setting data directory for advanced users</summary>
```

Comment:
```suggestion
<details><summary>Setting data directory permanently</summary>
```

Reply:
Not just for "advanced" users: for anyone who wants to have a persistent place to store data across sessions without having to set it manually each time. A large proportion of users.